android/src/main/res/values/strings.xml

```diff
@@ -195,4 +195,5 @@
 \n<li>New testing infrastructure for developers</li>
 \n<li>Simulation mode added to the developer options.</li>
 </string>
+    <string name="limit_map_zoom">Limit map zoom</string>
```

Comment:
Let's try that and see how it goes. I think this will get moved to Settings if it is liked.